echoregions/lines/lines.py

```diff
@@ -243,18 +236,17 @@ def filter_bottom(bottom, start_date, end_date):
                                 inside, outside."
                 )
 
-            # Interpolate on the sonar coordinates. Note that nearest interpolation has a problem
-            # when points are far from each other.
-            try:
-                bottom_interpolated = (
-                    filtered_bottom.reindex(joint_index)
-                    .interpolate(method=method, limit_area=limit_area)
-                    .loc[sonar_index]
-                ).fillna(max_depth)
-            except Exception as e:
-                # For all other such errors that may not necessarily deal with
-                # the specific values of method and limit_area.
-                print(e)
+            # Interpolate on the sonar coordinates. Note that nearest interpolation
+            # has a problem when points are far from each other.
+            # TODO There exists a problem where when we use .loc prior to reindexing
+            # we are hit with a key not found error. Potentially investigate this
+            # after major changes have been completed.
```

Comment:
What do you mean by major changes? 

Also I think `sonar_index` should be renames to `sonar_ping_time` to better describe what it is.

Reply:
Major changes referred to the refactoring.

Sounds good. I just did the renaming.